ada_2_fhir-r4/cio/2.0.0/payload/cio-Condition.xsl

```diff
@@ -146,19 +147,57 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     </xsl:call-template>
                 </xsl:for-each>
                 
-                <xsl:for-each select="diagnose_steller/*">
+                <xsl:for-each select="($hypersensitivityIntolerance | $reaction)/diagnose_steller/*">
                     <xsl:call-template name="makeReference">
                         <xsl:with-param name="in" select="."/>
                         <xsl:with-param name="wrapIn" select="'asserter'"/>
                         <xsl:with-param name="profile" select="$profileNameHealthProfessionalPractitionerRole"/>
                     </xsl:call-template>
                 </xsl:for-each>
+                
+                <evidence>
+                    <symptom>
+                        <detail>
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="../symptoom/relatie_aandoening_of_gesteldheid"/>
+                                <xsl:with-param name="wrapIn" select="'relatieSymptoom'"/>
```

Comment:
This parameter is used to indicate the FHIR element in which the reference should be made. In this case however the `.evidence.detail` element is already specified on line 160, so the `wrapIn` parameter is redundant here.